src/components/seeds/database/Filters.tsx

```diff
@@ -171,6 +179,27 @@ export default function Filters(props: Props): JSX.Element {
     );
   };
 
+  const preExpFilterPill: PillListItem<string> = filters.preExpFilter && {
+    id: 'preExpFilter',
+    label: strings.STATUS,
+    value: filters.preExpFilter.values.join(', '),
+  };
+
+  const filterPillItems = [];
+  if (preExpFilterPill) {
+    filterPillItems.push(preExpFilterPill);
+  }
+  for (const col of columns) {
+    const filter = filters[col.key];
+    if (filter) {
+      filterPillItems.push({
+        id: col.key,
+        label: col.name,
+        value: filter.values.join(`, `),
+      } as PillListItem<string>);
+    }
+  }
```

Comment:
suggestion - if it helps with perf.. move the filterPillItems into a useMemo?

Reply:
That's a pretty good idea I think 👍